addons/udes_stock/tests/plot_load_tests.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from scipy.stats import spearmanr
+from collections import defaultdict
+from matplotlib import pyplot as plt
+import sys
+
+COEFFICIENT_THRESHOLD = 0.7
+PVAL_THRESHOLD = 0.001
+
+# Do we want to bootstrap this?
+def scaling_coefficient(x, y):
+    """Nuno's neat method"""
+    dydx = np.diff(y)/np.diff(x)
+    coefficient, pval = spearmanr(x[1:], dydx)
+    area = np.sum(dydx - dydx[0])
+    return area, coefficient, pval
+
+
+def parse_time_file(filename):
```

Comment:
What file format are we reading here, and is there some pre-existing parser that would already handle it?

Reply:
It was just a text file with
   name of the function, the dependant variably and the time in a tab separated line 
   with a line of ='s to separate run for someone looking by eye 
Could probably just write it as a csv and use the csv reader instead. 
I'll change both the read and write of the file.